src/T2DMap.cpp

```diff
@@ -1990,40 +2009,40 @@ void T2DMap::createLabel(QRectF labelRect)
     mHelpMsg.clear();
 
     QMessageBox msgBox;
-    msgBox.setText("Text label or image label?");
+    msgBox.setText(tr("Text or Image label?"));
```

Comment:
Well it helps to pick out that there are two options and what those are - and linguistically it might help in translation - imaging you are Zaphod Beeblebrox in Zarniwoop's office: do you leave via the __Door__ or jump out the __Window__?

Reply:
Just door or window... these aren't names!

Reply:
☑️  Actually have changed it to "Type of label?" - which leads on nicely to the texts on the Buttons which are "Text Label" and "Image Label" as the answers to the question!